File: OMKARAUTOAPI/Consignment/Models/UnBillProvisionMstList.cs
using Shared.Models;

namespace Consignment.Models
{
    public class UnBillProvisionMstList
    {
        public List<UnBillProvisionMstModel> ProvisionList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: OMKARAUTOAPP/src/app/models/unbillprovisionmstlist.ts
import { Paginationmodel } from "./paginationmodel";
import { Unbilledprovisionmstmodel } from "./unbillprovisionmst";

export class UnbillprovisionmstList  {
    provisionList: Unbilledprovisionmstmodel [] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
